sirepo/template/activait.py

```diff
@@ -299,6 +299,38 @@ def stateful_compute_compute_column_info(data):
     return _compute_column_info(data.args.dataFile)
 
 
+def stateful_compute_sample_images(data):
+    import matplotlib.pyplot as plt
+    from base64 import b64encode
+    from pykern.pkcompat import from_bytes
```

Comment:
Use `from pykern import pkcompat` and then `pkcompat.from_bytes()`. You might also want to do the import at the top of the file, but I'm not sure of the protocol there.